pkg/lib/machine-dialogs.js

```diff
@@ -569,20 +569,15 @@
             var options = {};
             var match_problem = problem;
             var fp;
-            var path = machines.known_hosts_path;
 
             if (error_options) {
                 key = error_options["host-key"];
                 fp = error_options["host-fingerprint"];
-                if (error_options["invalid-hostkey-file"])
-                    path = error_options["invalid-hostkey-file"];
             }
 
             dialog.render({
                 'context_title' : dialog.context_title,
-                'path' : path,
                 'key' : fp,
-                'key_host' : key ? key.split(' ')[0] : null,
```

Comment:
The commit message should document the rationale for this (missing API in libssh to retrieve the file location). Could also be a separate commit for clarity.

Reply:
That make sense, I'll separate it.